docs/docs/dev_docs/tutorials/token_portal/depositing_to_aztec.md

```diff
@@ -0,0 +1,94 @@
+     - More on cancellers can be found in [this upcoming section](https://docs.google.com/document/d/1RNkwRrLZ74DkjowaEytjQfH0B1fbH4P5jc7jsr0tYuM/edit#heading=h.81q6cnz1h7jp)
+   - We encode this message as a mint_public function call, to specify the exact intentions and parameters we want to execute on L2.
+     - In reality the content can be constructed in any manner as long as the sister contract on L2 can also create it. But for clarity, we are constructing the content like a abi encoded function call.
+     - It is good practice to include all parameters used by L2 into this content (like the amount and to) so that a malicious actor can’t change the to to themselves when consuming the message.
+3. The tokens are transferred from the user to the portal using underlying.safeTransferFrom(). This puts the funds under the portal's control.
+4. Next we send the message to the inbox contract. It expects:
+   - recipient, a struct:
+     - the sister contract address on L2 that can consume the message.
+     - The version - just like Ethereum, there might be various versions, and upgrades of Aztec. By including a version, an ID, we can prevent replay attacks of the message on various aztec networks.
+   - Deadline by which the sequencer on L2 must consume the method. After this time, the message can be canceled by the “canceller”. We will implement this functionality later in the doc.
+   - A secret hash (fit to a field element). This is mainly used in the private domain and the preimage of the hash doesn’t need to be secret for the public flow. When consuming the message, one must provide the preimage. More on this when we create the private flow for depositing tokens.
+   - We also pass a fee to the sequencer for including the message. It is a uint64
+5. It returns a `bytes32 key` which is the id for this message in the Inbox.
+
+So in summary, it deposits tokens to the portal, encodes a mint message, hashes it, and sends it to the Aztec rollup via the Inbox. The L2 token contract can then mint the tokens when it processes the message.
+
+## Depositing tokens to Aztec privately
+
+Let’s do the similar for the private flow:
+
+#include_code deposit_private /l1-contracts/test/portals/TokenPortal.sol solidity
+
+Here we want to send a message to mint tokens privately on Aztec! Some key differences from the previous method are:
+
+- The content hash uses a different function name - `mint_private`. This is done to make it easy to separate concerns. If the contentHash between the public and private message was the same, then an attacker could consume a private message publicly!
+- Since we want to mint tokens privately, we shouldn’t specify a to Aztec address (remember that Ethereum is completely public). Instead, we will use a secret hash - `secretHashForRedeemingMintedNotes`. Only he who knows the preimage to the secret hash can actually mint the notes. This is similar to the mechanism we use for message consumption on L2
```

Comment:
```suggestion
- Since we want to mint tokens privately, we shouldn’t specify a `to` Aztec address (remember that Ethereum is completely public). Instead, we will use a secret hash - `secretHashForRedeemingMintedNotes`. Only he who knows the preimage to the secret hash can actually mint the notes. This is similar to the mechanism we use for message consumption on L2
```